src/nostr-post/nostr-post.ts

```
@@ -0,0 +1,825 @@
+  private stats: Stats | null = null;
+  private embeddedPosts: Map<string, NDKEvent> = new Map();
+
+  private receivedData: boolean = false;
+
+  private author: NDKUserProfile | null | undefined = {
+  name: '',
+  image: '',
+  nip05: '',
+  };
+
+  private onClick: Function | null = null;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)